rpc/client.go

```diff
@@ -59,28 +57,6 @@ func NewRpcClient(logger zerolog.Logger, trans transport.Requester) (*RpcClient,
 	return c, nil
 }
 
-// NewHttpRpcClient creates a new RpcClient using an http transport
-func NewHttpRpcClient(rpcServerUrl string) (*RpcClient, error) {
```

Comment:
We currently use this constructor in our[ boost integration](https://github.com/statechannels/boost/blob/f7d1b8ec9c98e3e4b72ed5c4ff032c279fcf9438/cmd/booster-http/server.go#L90). It is nice to have one constructor that does everything instead of manually constructing the transport

Reply:
Good call!

Reply:
`NewHttpRpcClient` method is refactored in https://github.com/statechannels/go-nitro/pull/1451/commits/957558706a840b891fc8457458d4a2b4789e2fd1